jvcl/run/JvTFGlance.pas

```diff
@@ -2847,8 +2847,12 @@ procedure TJvTFCustomGlance.CheckApptHint(Info: TJvTFGlanceCoord);
   Handled: Boolean;
 begin
   if Assigned(FViewer) and FViewer.ShowSchedNamesInHint then
+  begin
     ExtraDesc := StringsToStr(SchedNames, ', ', False);
-  ExtraDesc := ExtraDesc + #13#10;
+    ExtraDesc := ExtraDesc + #13#10;
+  end
+  else
+    ExtraDesc := '';
```

Comment:
The coding style says that a if a branch of an `if` statement requires the `begin`/`end` pair, then the other should have it as well.